src/exercises/02-Fragment-Container/README.md

```diff
@@ -6,13 +6,13 @@
 
 _Source: [Relay docs: Fragment Container](https://relay.dev/docs/v10.1.3/fragment-container/)_
 
-Building a React app requires intentional effort to isolate components. Markup, styles, state management, etc. typically belong to the nearest component that needs them. 
+Building a React app requires intentional effort to isolate components. Markup, styles, state management, etc. typically belong to the nearest component that needs them.
 
-Relay allows us to also precisely define the _data_ required to render each component in a tree, with Fragment Containers. 
+Relay allows us to also precisely define the _data_ required to render each component in a tree, with Fragment Containers.
 
-If a QueryRenderer is the top-level component of a tree that renders data, Fragment Containers define the contract each child component needs from the data. While there is a single QueryRenderer associated with each network request in our app, each request will retrieve the data for many Fragment Containers. 
+If a QueryRenderer is the top-level component of a tree that renders data, Fragment Containers define the contract each child component needs from the data. While there is a single QueryRenderer associated with each network request in our app, each request will retrieve the data for many Fragment Containers. Imagine the QueryRenderer as the tree and the Fragment Container as its branches.
```

Comment:
Was thinking the tree and branch analogy might fits nice here to illustrate the relationship between QueryRenderer and FragmentContainer.